include/nbl/builtin/hlsl/workgroup/fft.hlsl

```diff
@@ -103,6 +107,197 @@ void FFT(NBL_REF_ARG(Accessor) sharedmemAccessor, NBL_REF_ARG(complex_t<Scalar>)
     }
 }
 
+*/
+
+// ----------------------------------------- ABOVE MARKED FOR DELETION -----------------------------------------------------
+
+template<uint16_t K, bool Inverse, typneame Scalar, class device_capabilities=void>
+struct FFT;
+
+// For the FFT methods below, we assume:
+//      - Accessor is a global memory accessor to an array fitting 2 * _NBL_HLSL_WORKGROUP_SIZE_ elements of type complex_t<Scalar>, used to get inputs / set outputs of the FFT
```

Comment:
you only need `NBL_HLSL_WORKGROUP_SIZE` complex elements
cause you only trade half of your entries at every step

Reply:
update the comment